### PR TITLE
auth_proxy: Fix forgot_password url.

### PR DESCRIPTION
**Pull request by yellottyellott**
Practicing creating releases.

@buzzfeed/platform-infra

### CODE REVIEW

File: oauthproxy_test.go [hunk 1]
```diff
@@ -342,6 +342,8 @@ func TestForgotPasswordPage(t *testing.T) {
 	req, _ := http.NewRequest("GET", "/oauth2/forgot_password", nil)
 	proxy.ServeHTTP(rw, req)
 	assert.Equal(t, 200, rw.Code)
+	match, _ := regexp.MatchString(".*Community user login reset instruction.*", rw.Body.String())
+	assert.Equal(t, match, true)
```
Comment (mccutchen) [inline oauthproxy_test.go]: While we're in here, should we assert that the expected URL is present?

File: oauthproxy_test.go [hunk 2]
```diff
@@ -338,10 +338,21 @@ func TestForgotPasswordPage(t *testing.T) {
...
+
+	rw = httptest.NewRecorder()
+	req, _ = http.NewRequest("GET", "/oauth2/sign_in", nil)
+	proxy.ServeHTTP(rw, req)
+	assert.Equal(t, 200, rw.Code)
+	match, _ = regexp.MatchString("href=.*/forgot_password.*", rw.Body.String())
```
Comment (yellottyellott) [inline oauthproxy_test.go]: @mccutchen How's this?

Comment (mccutchen) [inline oauthproxy_test.go]: In the test, is it safe to hard-code `/oauth2/forgot_password`?  Given the URL just above, it seems like we can expect this specific URL to be in the output.

Comment (yellottyellott) [inline oauthproxy_test.go]: Yep, that turned out to be right!

**Comment by yellottyellott** [comment]
@mccutchen Let's get this out this morning.

**Comment by mccutchen** [comment]
👍

**Pull request merged**